src/main/java/org/mule/tools/devkit/sonar/utils/PomUtils.java

```diff
@@ -54,6 +64,36 @@ public static boolean isDevKitConnector(MavenProject mavenProject) {
         return parent != null && parent.getGroupId().equals(DEVKIT_GROUP_ID) && (parent.getArtifactId().equals(DEVKIT_ARTIFACT_ID) || parent.getArtifactId().equals(CERTIFIED_DEVKIT_ARTIFACT_ID));
     }
 
+    public static VersionUtils setLatestVersion() {
+        VersionUtils latestVersion = new VersionUtils("3.0.0");
```

Comment:
It's the lowest version that there is, it would be like setting a variable to 0 while finding for the greatest number. I can use "0.0.0", or maybe even better, set it with the first value of the list